src/frameworks/uikit/ui_color.rs

```diff
@@ -91,6 +91,18 @@ pub const CLASSES: ClassExports = objc_classes! {
 
 // TODO: more initializers, set methods, more accessors
 
+- (id)initWithWhite:(CGFloat) w alpha:(CGFloat) a {    
+    if w < 0.0 {
```

Comment:
Should there not be similar clamping for the alpha part too?

By the way, you can use Rust's `.clamp()` method on floats.

Reply:
I don't if it should be clamped because in the documentation it only mentions the white arg being clamped "white values below 0.0 are interpreted as 0.0, and values above 1.0 are interpreted as 1.0."

Reply:
What documentation are you reading? [developer.apple.com says](https://developer.apple.com/documentation/uikit/uicolor/1621944-initwithwhite):

> Alpha values below 0.0 are interpreted as 0.0, and values above 1.0 are interpreted as 1.0.

Reply:
I'm an idiot I will make this fix.